app/views/obituaries/show.html.erb

```diff
@@ -1,5 +1,5 @@
 <div class="obituary-image">
-  <%= image_tag(@obituary.image_url) %>
+  <%= image_tag(@obituary.image_url) rescue image_tag 'https://image.shutterstock.com/image-vector/ui-image-placeholder-wireframes-apps-260nw-1037719204.jpg' %>
```

Comment:
This will stop the page from throwing and error and use the default image if the user entered an image path that will cause an error

Reply:
Feel free to change the default image if anyone has a better one!

Reply:
Once we get the logo nailed down will drop that in here, if everyone is on board.

Reply:
I think that makes sense @stellakunzang

Reply:
nice work here